packages/ripple-nuxt-tide/lib/core/utils/site-section.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Get the site section data
+ * @param {string|number} sectionId
+ * @param {Object[]} sites - Tide field_node_site data
+ * @returns {Object} site section data
+ */
+const getSiteSectionData = (sectionId, sites) => {
+  sectionId = parseInt(sectionId)
+  const site = sites.find(site => site.drupal_internal__tid === sectionId)
+  if (site) {
+    return {
+      id: sectionId,
+      name: site.name,
+      quickexit: site.field_site_show_exit_site
```

Comment:
I'm assuming this will always have a value. Or do we need to add defensive or `|| false`?

Reply:
@rubyllamadora good pick! It will be `null`, fixed so we can make sure it's a boolean.